### PR TITLE
3.11 release notes - clarify ansible version

### PR DESCRIPTION
**Pull request by vrutkovs**
Ansible 2.6 minimum is required, ansible 2.7 is not supported

### CODE REVIEW

File: release_notes/ocp_3_11_release_notes.adoc [hunk 1]
```diff
@@ -228,6 +228,8 @@ $ subscription-manager repos --enable="rhel-7-server-rpms" \
     --enable="rhel-7-server-ansible-2.6-rpms"
 ----
 
+Installation with Ansible 2.7 is not yet supported.
```
Comment (kalexand-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: s/Installation with/

**Review by kalexand-rh — COMMENTED** [review]
@vrutkovs, thanks! I have one request, then I'll merge.

File: release_notes/ocp_3_11_release_notes.adoc (hunk 1 above)
Comment (vrutkovs) [reply release_notes/ocp_3_11_release_notes.adoc]: Fixed

**Review by kalexand-rh — APPROVED** [review]
:tada:

**Pull request merged**